pages/linux/snap.md

```diff
@@ -0,0 +1,24 @@
+# snap
+
+> Tool for managing the *snap* self-contained softwawre packages.
```

Comment:
Please remove the markdown formatting `**` from snap. Use backticks around "apt" and double quotes around ".deb".

Reply:
Submitted it, please check it again; made other changes too taking into consideration the other comments.

Thank you.